Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2765 Pt. III: Add New Delivery Systems Validations

### Description

Adding two (2) new validations for delivery systems and their nested selections to FFY2023 measures

### Related ticket(s)


MDCT-2765

---
### How to test

- Log in as state user
- Navigate to FY2023
- Do not select an option for `Which delivery systems are represented in the denominator?`, like so:
 
![Screenshot 2023-08-01 at 10 36 07 AM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/450b917e-c4e0-4954-b593-d7c621bf9c6d)

Validate the measure and you should see this error message:

![Screenshot 2023-08-01 at 10 36 30 AM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/b34fc94f-89af-41e5-8ccd-2ab3925abec1)

- Next, select an option for that same question but don't select anything for the nested responses, like so:

![Screenshot 2023-08-01 at 10 36 16 AM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/1a7b2033-d18a-4633-ab00-a943feeb63f1)

Validate the measure and you should see this other error message:

![Screenshot 2023-08-01 at 10 37 18 AM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/3ca7d83d-22c3-46f0-914c-9d87be24d573)

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
